src/styles/components/buttons/styles.less

```diff
@@ -102,149 +107,132 @@ components/buttons.less
         &:last-child {
           margin-bottom: 0;
         }
-      }
-    }
-  }
 
-  /* -----------------------------------------------------------------------------
-  --------------------------------------------------------------------------------
+      }
 
-  Button Group
+    }
 
-  --------------------------------------------------------------------------------
-  ----------------------------------------------------------------------------- */
+  }
 
   .button-group {
-
     margin-bottom: @base-spacing-unit * 0.5;
```

Comment:
We should not need this style if we are using `.button-collection` to wrap a `.button-group`.